### PR TITLE
Tests: Calling .rules() on an empty jQuery object results in an error.

### PR DESCRIPTION
**Pull request by staabm**
Refs #1706

### CODE REVIEW

**Comment by staabm** [comment]
@whefter could you confirm the fix?

File: test/rules.js [hunk 1]
```diff
@@ -396,3 +396,10 @@ test( "rules(), normalizer", function() {
 		return err.name === "TypeError" && err.message === "The normalizer should return a string value.";
 	}, "This should throw a TypeError exception." );
 } );
+
+test( "rules() - on empty jquery set #1706", function() {
+	var element = $( "#firstname .mynonexistantclass" );
+
+	var result = element.rules( "add", "whatever" );
+	equal( result, null, "can work on an empty set without a js error" );
```
Comment (Arkni) [inline test/rules.js]: I guess using `strictEqual` is more appropriate here:
```js
strictEqual( result, undefined, "can work on an empty set without a js error" );
```

**Comment by Arkni** [comment]
A rebase is needed here.

**Comment by staabm** [comment]
rebased and adjusted per feedback @Arkni.

just leveled my git knowledge up, yay :100:

**Comment by Arkni** [comment]
:+1:

**Pull request merged**